docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,387 @@
+smart card authentication.
+
+### Export Teleport CA and CRL.
+
+The following step requires an existing cluster. If you don't already have a
+Teleport cluster up and running, see our general [Getting
+Started](../../try-out-teleport/introduction.mdx) guide.
+
+<Admonition type="note" title="Database CA Rotation">
+  These steps will need to be repeated if Teleport's database certificate
+  authority is rotated.
+</Admonition>
+
+1. Get the Teleport database CA certificate by running:
+
+```code
+$ tctl auth export --type=db-der > db-ca.cer
+```
+
+2. Get the Teleport database CRL by running:
+
+```code
+$ tctl auth crl --type=db > db-ca.crl
+```
+
+3. Transfer the `db-ca.cer` and `db-ca.crl` files to a Windows machine where you can manage your group policy.
+
+### Create a GPO and import the Teleport CA.
+
+<Admonition type="note" title="Domain Wide Policy">
+  For the purposes of this guide, we apply the GPO we are about to create to our
+  entire AD domain. In the case where you wish for only a subset of computers
+  within your AD domain to be accessible via Teleport, you should apply the GPO
+  to an OU that includes only such computers.
+</Admonition>
+
+1. Create a GPO with name like `Teleport DB Access`.
+
+```powershell
+$GPOName="Teleport DB Access"
+New-GPO -Name $GPOName | New-GPLink -Target $((Get-ADDomain).DistinguishedName)
+```
+
+2. Again open the `Group Policy Management` program, and on the left pane,
+   navigate to `$FOREST > Domains > $DOMAIN > Group Policy Objects`.
+3. Right click on the GPO you just made (`Teleport DB Access`), and select `Edit...`.
+4. In the group policy editor, select:
+
+```text
+Computer Configuration > Policies > Windows Settings > Security Settings > Public Key Policies
+```
+
+5. Right click on `Trusted Root Certification Authorities` and select `Import`.
+6. Click through the wizard, selecting your CA file (`db-ca.cer`).
+
+<Figure align="left" bordered caption="Import Teleport CA">
+  ![Import Teleport CA](../../../img/desktop-access/ca.png)
+</Figure>
+
+### Enable smart card service.
+
+Teleport performs certificate based authentication by emulating a smart card.
+
+1. Still editing your `Teleport DB Access`, select:
+
+```text
+Computer Configuration > Policies > Windows Settings > Security Settings > System Services
+```
+
+2. Double click on `Smart Card`, select `Define this policy setting` and switch
+   to `Automatic` then click `OK`.
+
+<Figure align="left" bordered caption="Enable the Smart Card Service">
+  ![Enable Smartcard](../../../img/desktop-access/smartcard.png)
+</Figure>
+
+<Admonition type="note" title="gpupdate.exe">
+  You will be modifying GPOs, and sometimes GPO modifications can take some time
+  to propagate to all hosts. You can force your changes to take effect
+  immediately on your current host at any time by opening a PowerShell prompt
+  and running `gpupdate.exe /force` (though their effects may still take time to
```

Comment:
What does "their" refer to here? All of the GPOs you modified in this guide?

Reply:
Yes. Should we change it to make it more clear? Also, in this guide, we're only working with a single GPO.